crl-janitor/pubsub.tf

```diff
@@ -48,5 +63,30 @@ resource "google_pubsub_subscription_iam_member" "crl_janitor_client_can_manage"
 
   subscription = google_pubsub_subscription.crl-janitor-pubsub-subscription[0].name
   role = "roles/editor"
-  member   = "serviceAccount:${google_service_account.app[0].email}"
+  member = "serviceAccount:${google_service_account.app[0].email}"
+}
+
+// Pubsub creates a SA account for each project: service-project-number@gcp-sa-pubsub.iam.gserviceaccount.com for
+// deal lette queue message management.
+// See https://cloud.google.com/pubsub/docs/dead-letter-topics#granting_forwarding_permissions.
+data "google_project" "project" {
```

Comment:
Just checking my understanding of this stanza (based on [these docs](https://www.terraform.io/docs/configuration/data-sources.html#using-data-sources)): we are asking the Google provider to load data related to the current Google project into a data resource namespace variable 'project'... right?

How does Terraform know which "project" to load here? Is this set globally somehow, or derived from some deployment / provider context? Or somehow extracted from var.google_project?

Reply:
I think by default it uses the default provider project, for example, the value is here: 
https://github.com/broadinstitute/terraform-ap-deployments/blob/bf6fd9e30c801b222e11e296380fe0c76e94a59a/terra-env/tfvars/tools.tfvars#L2